.github/workflows/release.yaml

```diff
@@ -226,7 +73,7 @@ jobs:
         run: git checkout -b "docs-for-release-${GITHUB_REF#refs/*/}"
 
       - name: Merge with main
-        run: git merge origin/main --no-ff -Xtheirs -m "Merge branch 'main' into temporary docs branch"
+        run: git merge origin/main --no-ff -m "Merge branch 'main' into temporary docs branch"
```

Comment:
Can you revert this change? This merge strategy is needed and the change doesn't belong here anyway.

Reply:
ouch something went wrong with merge conflicts. on it

Reply:
No problem! Thanks :)